pkg/metrics/session.go

```diff
@@ -227,6 +228,16 @@ func InitSessionMetrics() {
 			Name:      "transaction_fair_locking_usage",
 			Help:      "The counter of statements and transactions in which fair locking is used or takes effect",
 		}, []string{LblType})
+
+	// Moved from client-go module to tidb, to keep consistency with history versions, keep the subsystem name "tikvclient"
+	TiKVPessimisticLockKeysDuration = prometheus.NewHistogram(
+		prometheus.HistogramOpts{
+			Namespace: "tidb",
+			Subsystem: "tikvclient",
+			Name:      "pessimistic_lock_keys_duration",
+			Buckets:   prometheus.ExponentialBuckets(0.001, 2, 24), // 1ms ~ 8389s
```

Comment:
The default value of `innodb_lock_wait_timeout` is `50s`, the max bucket value could be reduced to hundreds of seconds to save buckets.

Reply:
Make sense, will change it.

Reply:
Done